content/features/proxy2/configure.md

```diff
@@ -0,0 +1,411 @@
+        },
+    },
+```
+
+To mark the backend as online again, remove the `_down_` or `down=true` attribute from the backend's definition in the proxy configuration file.
+
+## Prioritize a route using local zones {#zones}
+
+If you have organized your pools into named sets, then you can further configure the proxy to prefer routing requests to certain pools within a set, even if other pools are available. You might want to do this to prefer routing requests to a pool that is geographically closer to the proxy, for example.
+
+Doing this involves the following steps:
+
+1. Declare a _local zone_ in your proxy configuration file.
+1. Attach zone labels to the pools within your set definitions.
+1. In your route definitions, invoke a route handler that makes use of zone information. The only such handler defined by the standard route library is `route_zfailover`.
+
+To declare the local zone and attach zone labels to your defined pools, modify your configuration file's `pools{}` block like this:
+
+```lua
+local_zone("{{<var>}}LOCAL_ZONE_NAME{{</var>}}")
+
+pools {
+    {{<var>}}SET_NAME{{</var>}} = {
+    {
+        {{<var>}}POOL_ZONE_NAME{{</var>}} = { backends = {{<var>}}[ ... ]{{</var>}} },
+        {{<var>}}[ ... ]{{</var>}}
+    },
+    {{<var>}}[ ... ]{{</var>}}
+}
+```
+
+Replace the following:
+* <var>LOCAL_ZONE_NAME</var>: the name of the zone that this proxy prefers to route requests to.
+* <var>POOL_ZONE_NAME</var>: the name of the zone that this pool is in.
+
+The following example excerpt from a proxy configuration file that declares a local zone—in this case, "`west`"—and then defines two sets of zoned pools.
+
+```lua
+local_zone("west")
+
+pools {
+    main_pools = {
+        west   = {
+            backends = {
+                "192.0.2.1",
+                "192.0.2.2",
+                "192.0.2.3:11212",
+            },
+        south  = {
+            backends = {
+                {{<var>}}[ ... ]{{</var>}}
+            },
+        east   = {
+            backends = { 
+                {{<var>}}[ ... ]{{</var>}}
+            } 
+        },
+    },
+    
+    customer_pools = {
+        west   = {
+            backends = {
+                "203.0.113.1",
+                "203.0.113.10",
+                { host="203.0.113.20", port=11212, flaptime=10 },
+            },
+        south  = {
+            backends = {
+                {{<var>}}[ ... ]{{</var>}}
+            },
+        east   = {
+            backends = { 
+                {{<var>}}[ ... ]{{</var>}}
+            } 
+        },
+    },
+}
+```
+
+The only route handler that makes use of zone information is [`route_zfailover`]({{<proxy_base_path>}}reference#route_zfailover). The following proxy configuration excerpt extends the previous example with a `routes{}` block that uses `route_zfailover`.
+
+```lua
+routes {
+    map = {
+        main = route_zfailover {
+            children = { "main_pools" },
```

Comment:
`children = "set_main_pools"`

see: https://github.com/memcached/memcached-proxylibs/blob/main/lib/routelib/examples/zfailover.lua

Reply:
Fixed. I also similarly updated line 358. (And renamed the arguments to `main_pool_set` and `customer_pool_set` for reasons in previous comment.)